### DISCUSSION TITLE
Alineaciones de texto e imágenes

### DISCUSSION THREAD

**Discussion by EAleman8** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#302] [Q&A] [answered]
Que tal!

Estoy teniendo complicaciones con la práctica de CSS ya que no cuento con mucho tiempo para investigar por mi cuenta, quisiera saber si alguien me puede ayudar en saber cómo realizar este tipo de alineaciones hacia la izquierda y derecha en cada elemento y cómo alinear el texto en cada caso, no he logrado encontrar ejemplos comprensibles y estoy estancado porque no tengo ni idea de como empezar esto :,)

![imagen](https://user-images.githubusercontent.com/99446604/157126281-02ff9606-becd-49dd-b562-a028afc3af57.png)

**Comment by hackit92** [comment]
Hay muchas formas de trabajar en código asi que no hay una respuesta absoluta, pero por lo menos yo hice uso de flexbox y cssgrid a lo largo de la página según me convenía, sin embargo no es la única forma. Por ejemplo la navbar podria tener un display: inline y al logotipo añadirle una clase en css personalizada para hacer los ajustes necesarios o incluso tener 2 divs, es cuestión de como uses las herramientas que ya conoces y veas cuales otras puedes complementar.  

Échale un ojo a esto, seguro te ayuda https://freshman.tech/flexbox-navbar/

**Reply by EAleman8** [reply]
Justo estoy tratando de mejorar con flexbox, y gracias por el enlace, muy funcional!

**Comment by doguedogue** [comment, marked as answer]
La vieja confiable con una tabla y porcentajes en columnas.

Bootstrap, usa columnas. 

Ya con algo mejor flexbox o gridbox, las dos son geniales para organizar con css.

**Reply by EAleman8** [reply]
LE voy a dar una checada a bootstrap, espero que quede decente, gracias!
